### DISCUSSION TITLE
email notification of new version?

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1163] [Ideas]
I'm not someone to constantly check the website for a new release.

Why not create a mailing list where people could subscribe to get notifications of new versions and critical bugs (and possibly their workarounds)?

**Comment by vkbo** [comment]
I could possibly do that, but I don't really trust mailinglists to take people's privacy seriously and I don't want to do this manually.

In any case, you can check for a new release in "Help > Check for New Release" in the menu inside novelWriter. When you have a GitHub account, you can also select to be notified when there are new releases directly from the repository:

![image](https://user-images.githubusercontent.com/1619840/195623995-925e0699-1653-41b0-b1bf-4849595af5e0.png)
